front_end/panels/ai_chat/evaluation/framework/types.ts

```
@@ -160,6 +160,11 @@ export interface EvaluationConfig {
   evaluationModel: string;
   evaluationApiKey: string;
   
+  // Model settings for tools and agents under test
+  mainModel: string;
+  miniModel: string;
+  nanoModel: string;
+  
```

ESLint

[error] Trailing spaces not allowed.

(@stylistic/no-trailing-spaces)